### PR TITLE
Switch to downloading blobs from the Prysm RPC API

### PR DESCRIPTION
**Pull request by mdehoog**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -4,5 +4,5 @@
 	branch = eip-4844
 [submodule "prysm"]
 	path = prysm
-	url = https://github.com/Inphi/prysm
-	branch = eip-4844
+	url = https://github.com/mdehoog/prysm
+	branch = blobs-api
```
Comment (mdehoog) [inline .gitmodules]: switch this back once https://github.com/Inphi/prysm/pull/21 is merged

**Review by roberto-bayardo — APPROVED** [review]

**Comment by roberto-bayardo** [comment]
One thought though: would we want to keep both approaches for more thorough testing?  Or is most of that removed code duplicating what's now in the Prysm RPC API?

**Comment by Inphi** [comment]
> 

The removed code isn't duplicating the RPC API. I think we should keep and test both approaches. The `beacon-node-follower` receives sidecars from the `beacon-node` broadcast so it doesn't need to use the p2p RPC.

We can still merge this PR, as I'd be glad to get rid of that code, but only if we add a new test scenario that asserts p2p RPC works. For example, we setup a test with a late `beacon-node-follower`; which triggers initial-sync so it requests sidecars via p2p RPC. This is a good test to have anyways as intiial-sync is a pretty important codepath.

**Comment by Inphi** [comment]
This was implemented via https://github.com/Inphi/eip4844-interop/pull/44, which also adds a historical sync e2e test so we don't lose test coverage. Closing.

**Pull request closed without merging**